helpers/controllers/starsController.ts

```diff
@@ -17,6 +17,10 @@ export const setStar = async (
   try {
     const studentId = validateStudentId(req)
     const { lessonId, mentorId } = arg
+
+    if (studentId === mentorId) {
+      throw new Error('Unable to give error to yourself')
```

Comment:
Unable to give **ERROR** to yourself

🤔

Shouldn't it be star?

Reply:
Ohoho good catch :)